context.go

```diff
@@ -1312,6 +1312,35 @@ func (c *Context) addVariationDependency(module *moduleInfo, variations []Variat
 	}}
 }
 
+func (c *Context) addInterVariantDependency(origModule *moduleInfo, tag DependencyTag,
+	from, to Module) []error {
+
+	var fromInfo, toInfo *moduleInfo
+	for _, m := range origModule.splitModules {
+		if m.logicModule == from {
+			fromInfo = m
+		}
+		if m.logicModule == to {
+			toInfo = m
+			if fromInfo != nil {
+				return []error{&Error{
```

Comment:
Why an error here, and a panic below? Both are go logic errors, not user blueprint error (so the position isn't helpful)